TAO/docs/compiler.html

```diff
@@ -169,8 +198,47 @@ <h3>Generated Files</h3>
 &lt;orb.idl&gt; when it is included in an IDL file.  TAO_IDL will
 display a warning message when neither is defined.</P>
 
-<HR><P>
-<h3>Operation Demuxing Strategies</h3>
+<hr>
+<h2>IDL Preprocessor Macros</h2>
+
+<p>The preprocessor macros available in IDL include all the ones built-in to
+the preprocessor when it's being used for C/C++. In addition to the ones
+built-in to the preprocessor, the following are also defined by
+<code>tao_idl</code>:</p>
+
+<table>
+  <tr>
+    <th>Name</th>
+    <th>Description</th>
+  </tr>
+
+  <tr>
+    <td><code>__TAO_IDL</code></td>
+    <td>
+      <p>The version of ACE that <code>tao_idl</code> was built with. The format
+      is one byte for each version field in hex format. For example, in a
+      theoretical TAO version 3.2.1, which would correspond to ACE version
+      7.3.1, <code>__TAO_IDL</code> would be <code>0x070201</code>.</p>
+
+      <p>It's also possible to include <code>tao/Version.h</code> and use the
+      same macros from that header file as you can use in C++.</p>
```

Comment:
These could be different versions?  They probably shouldn't be though

Reply:
I agree that it's unlikely. Maybe if the user was somehow dealing with multiple versions of TAO or maybe more likely they updated `tao/Version.h`, but didn't recompile `tao_idl` or something like that. I think is later is already covered by the version guard.